src/main/java/io/github/terra121/util/CardinalDirection.java

```diff
@@ -0,0 +1,59 @@
+package io.github.terra121.util;
+public enum CardinalDirection {
```

Comment:
Formatting. Correct indentation and and some empty lines for better readability.